release/models/interfaces/openconfig-if-ip.yang

```diff
@@ -514,6 +520,14 @@ module openconfig-if-ip {
       description
         "The length of the subnet prefix.";
     }
+
+    leaf type {
+      type oc-inet:ipv6-address-type;
+      default GLOBAL_UNICAST;
+      description
+        "This ipv6 address is one of the ipv6-address-type values";
```

Comment:
I suggest we make this a clearer description. Something like:

```
Specifies the explicit type of the IPv6 address being assigned
to the subinterface. By default, addresses are assumed to be
global unicast; where a link-local address is to be explicitly
configured, this leaf should be set to LINK_LOCAL.
```

I think we want to be clear how a leaf should be used.

Can we also add a TODO for a feature profiles test for this too?

Reply:
TODO: https://github.com/openconfig/featureprofiles/issues/1108